src/redux/books/booksSlice.js

```diff
@@ -1,24 +1,84 @@
-import { createSlice } from '@reduxjs/toolkit';
-import bookList from './bookList';
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const url = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/hNgqcuwZELgkJT0cAZJB/books';
+
+export const postBook = createAsyncThunk(
+  'books/postBook',
+  async (newBook) => {
+    const response = await axios.post(url, newBook);
+    return response.data;
+  },
+);
+
+export const getBooks = createAsyncThunk(
+  'books/getBooks',
+  async () => {
+    const response = await axios(url);
+    return response.data;
+  },
+);
+
+export const deleteBook = createAsyncThunk(
+  'books/deleteBook',
+  async (id) => {
+    const response = await axios.delete(url + id);
+    return response.data;
+  },
+);
```

Comment:
- Kindly, you are using the incorrect URL for delete, the URL is `url/id` you missed to add `/` between, so it is returning `404`, please add `/` and fix it 👍👌

Reply:
Thanks for the information. 👍